client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -492,6 +492,21 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         MOBILE_APP_LOCALE("mobileAppLocale", "en_US", false),
 
         /**
+         * This parameter represents the appium device.
+         */
+        MOBILE_DEVICE("mobileDevice", "", false),
+
+        /**
+         * This parameter represents the appium device platform version.
+         */
+        MOBILE_PLATFORM_VERSION("mobilePlatformVersion", "", false),
```

Comment:
Let's be consistent with the `MobileTest` annotation and use `mobileDevice={device}:{platformVersion}`... This `ConfigProperty` can go away. 

`MobileTestSession` might need to be updated to support this change.